<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HumanPlayerGUIClass</class>
 <widget class="QDockWidget" name="HumanPlayerGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>136</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kontrolle</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelPlayer">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelPhase">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonNextPhase">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>-&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label1">
      <property name="text">
       <string>Angreifer:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label4">
      <property name="text">
       <string>Nachschubland:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelAtt">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label2">
      <property name="text">
       <string>Verteidiger:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelDef">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label3">
        <property name="text">
         <string>Armeen:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditArmys">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>72</width>
        <height>188</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Karten:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidgetCards">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>80</height>
       </size>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonExchange">
      <property name="text">
       <string>Eintauschen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>listWidgetCards</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>HumanPlayerGUIClass</receiver>
   <slot>slotCardSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonExchange</sender>
   <signal>clicked()</signal>
   <receiver>HumanPlayerGUIClass</receiver>
   <slot>slotExchangeCards()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>474</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotCardSelectionChanged()</slot>
  <slot>slotExchangeCards()</slot>
 </slots>
</ui>
